controller/src/main/java/io/pravega/controller/server/eventProcessor/requesthandlers/DeleteReaderGroupTask.java

```diff
@@ -55,7 +55,7 @@ public DeleteReaderGroupTask(final StreamMetadataTasks streamMetaTasks,
       UUID readerGroupId = request.getReaderGroupId();
       long generation = request.getGeneration();
       final RGOperationContext context = streamMetadataStore.createRGContext(scope, readerGroup);
-      return streamMetadataStore.getReaderGroupId(scope, readerGroup, context, executor)
+      return streamMetadataStore.getReaderGroupId(scope, readerGroup)
```

Comment:
why are we removing the context?

Reply:
getReaderGroupId() method does not use those parameters though they are being passed.